### PR TITLE
Custom options

### PR DESCRIPTION
**Pull request by aarya123**
Gives handlers the power to add custom options

### CODE REVIEW

File: magick.py [hunk 1]
```diff
@@ -490,13 +509,13 @@ def convert(self, path, chunk_ready=None, complete=None, error=None):
                 return
 
         command = self.convert_cmdline(path, source is not None)
-        logger.debug("CONVERT %s (opts: %s)" % (path, repr(self.options)))
+        logger.debug("CONVERT %s (opts: %s) COMMAND %s" % (path, repr(self.options), command))
```
Comment (strattonbrazil) [inline magick.py:512]: Nice to have the command explicitly printed out.

**Comment by strattonbrazil** [comment]
Looks great.  Did you want me to merge it or was someone else going to review it as well?

**Comment by aarya123** [comment]
I had anil look at it as well.
I'll merge it since I'm going to update the internal version as well. Thanks though! :)

**Comment by aarya123** [comment]
Everything seems good to go!

**Pull request merged**